pages.id/osx/brightness.md

```diff
@@ -0,0 +1,15 @@
+# brightness
+
+> Menampilkan dan mengubah brightness (tingkat kecerahan) untuk internal dan external displays.
+
+- Menampilkan brightness sekarang:
+
+`brightness -l`
+
+- Mengubah brightness menjadi 100%::
```

Comment:
This looks like a typo in the main page too. Could you fix it in this PR?
```suggestion
- Mengubah brightness menjadi 100%:
```

Reply:
I see, on it